src/lattices.jl

```diff
@@ -377,37 +377,21 @@ normscale(flat::ModulatedFourierLattice, expon::Real) = normscale!(deepcopy(flat
 
 In-place equivalent of `normscale`: changes `flat`.
 """
-function normscale!(flat::ModulatedFourierLattice, expon::Real)
-    if !iszero(expon)
+function normscale!(flat::ModulatedFourierLattice, Gs::Union{ReciprocalBasis{D}, Nothing} = nothing, expon::Real) where D
```

Comment:
The `Gs` argument has to be the last argument in order to have a default value:
```suggestion
function normscale!(flat::ModulatedFourierLattice,
                    expon::Real,
                    Gs::Union{ReciprocalBasis{D}, Nothing} = nothing) where D
```

Reply:
I see you did this before I actually managed to get the comments "published" (always find the commenting interface fiddly...)